assets/src/constants.js

```diff
@@ -66,6 +70,31 @@ export const BLOCK_ICONS = {
 	'amp/amp-story-page': <svg id="story-page-icon" viewBox="0 0 24 24"><g id="icon" fill="#181D21"><path id="page" d="M18.4 21H5.6V3h7.8l5 4.9V21zM7.1 19.5h9.8V8.6l-4-4.1H7.1v15z" /><path id="corner" d="M11.5 5.4v4.3h4.4" /></g></svg>,
 };
 
+export const ICONS = {
```

Comment:
Since we're adding some new icons here, can we actually move all these (including the one from `BLOCK_ICONS`) to `assets/images` and import them here like we do with the font name SVGs? 

This way we benefit from the SVG loader in the webpack config, which automatically optimizes the SVGs too.

Reply:
Saw that you already did that, thank you!